### ISSUE TITLE
Missing "Copy as plain text" button.

### ISSUE THREAD

**Issue by GH-User** [emvaized/selecton-extension#80]
Apologies, I did not find the "Copy as plain text" button. How can I activate it?
I'm sure it is there, I don't think, the creator forgot such a very basic function.

**Comment by emvaized** [comment]
No, it is not there. It's not so much that I forgot about it, but rather that I never needed it. And surprisingly, for such a very basic function, you are the second person over 3 years who mentioned it.

**Comment by GH-User** [comment]
Thank you for your quick answer. It's interesting, currently I'm testing many "Auto Copy" extensions for Chromium. Approx. 80% of all tested extensions do have a "copy as plain text" option. Your extension has so many options, I just couldn't imagine that such a fundamental function (for me) was missing. Very often I need copied text without any formatting. Is it possible for you to add such a function?

**Comment by emvaized** [comment]
I will think about adding this option in future updates. Probably it could utilize the middle or right button click on the "Copy" button, as it currently does nothing.
